cmd/heartbeat/heartbeat.go

```diff
@@ -76,6 +77,15 @@ func SendHeartbeats(v *viper.Viper, queueFilepath string) error {
 	setLogFields(params)
 	log.Debugf("params: %s", params)
 
+	if RateLimited(params.Offline) {
+		if err := offlinecmd.SaveHeartbeats(v, nil, queueFilepath); err != nil {
```

Comment:
linter is warning with
```
early-return: if c { ... } else { ... return } can be simplified to if !c { ... return } ... (move short variable declaration to its own line if necessary) (revive)
```

Reply:
That's how I had it previously

Reply:
I'll change it back, but we need the variable to be named other than `err` and not scoped to the `if` block.